tests/test_helpers.py

```diff
@@ -17,6 +17,19 @@
       "--date", "2020-04-29", "-c", "afriend", "-d", "more work",
       "-m", "ascopatz", "-g", "SymPy-1.1"],
      "20ly_newprojectum has been added in projecta\n"),
+    (["helper", "a_proposal", "an empty new proposal","100.0","To understand numbers"],
+    "20_an_empty_new_proposal has been added in proposals\n"),
+    (["helper", "a_proposal", "a sparse new proposal","500.0","To destroy numbers",
```

Comment:
something I don't expect here is that an optional argument has been stated but no value given. This is not quite the behhavior I want, so if this is what our use of defaults is giving maybe we need to walk it back again.  I kind of want that if a flag is not specified it gets a default, but if it is specified it should take a value.      I may have just caused you lots of unnecessary work and we have to go back to the if ``rc.something`` wrapping....sorry

Reply:
will do! luckily, my commits can save me here

Reply:
👍